collections/_article/godotcon-2023-cfp.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Godot Conference 2023: Call for Participations"
+excerpt: "GodotCon 2023 needs participations"
+categories: ["events"]
+author: Julian
+image: /storage/blog/covers/godotcon-2023.webp
+image_caption_description: Image by Raffaele Picca
+date: 2023-08-06 17:00:00
+---
+
+We are organizing the annual Godot Conference that will happen [live in Munich](/article/godotcon-2023/) on November 4th and 5th 2023 for the 
+first time since [the last occurrence in 2020](/article/meet-community-fosdem-and-godotcon-2020/). We are really thrilled to meet again fellow Godot users and developers to share experiences during the two days of the event.
+
+## Call for speakers & workshop
+
+We would like to propose all kinds of presentations during the event, and we need you for this! If you are interested in 
+sharing your project or experience that could interest Godot users, feel free to propose a dedicated talk or workshop on [our dedicated platform](https://events.godotengine.org/).
+
+We are opening 2 tracks: 
+
+* Main track
+* Workshops
+
+Please note that depending on the number of proposals we receive and the number of available slots, we may have to reject some proposals. Do not feel left out! You may propose again next year.
+
+### Main track
+
+This track is open to talks for various topics. We would like to see anything Godot-related or even not directly related to Godot but still relevant for game or engine developers.
+
+This track lasts the two whole days of the event. We propose two talk durations: **20 minutes or 50 minutes (+ 5-10 min Q&A)**, though if you need a different duration, you may express your needs in your proposal. When possible, we will change your talk duration accordingly.
```

Comment:
These times are different from what is listed on events.godotengine.org. Does this article need to be updated or does events.godotengine.org need to be updated?

Reply:
events.godotengine.org proposes 30 minutes and 60 minutes talks. These correspond to maximum possible lengths 20+10 and 50+10.

Though we could change this article to `30 minutes or 60 minutes (**including** 5-10 min Q&A)`

Reply:
Ah that makes sense. Either is fine. I'm happy to leave it as-is too

Reply:
Ah that makes sense. Either is fine. I'm happy to leave it as-is too

Reply:
Changed nevertheless for the sake of clarity ;)